src/components/game/Sidebar/Player.js

```diff
@@ -53,7 +54,7 @@ const Player = ({
                 duration={200}
                 verticalAlign="middle"
               >
-                {actions.slice(0, 3).map((action, idx) => (
```

Comment:
Are you sure about this? 
![Screenshot 2020-04-08 at 17 53 33](https://user-images.githubusercontent.com/2972266/78805626-ed978280-79c1-11ea-8d40-7bfdbde06672.png)

Reply:
I removed it for debugging 🙈